### ISSUE TITLE
pubsub emulator: support message ordering for automated subscriptions

### ISSUE THREAD

**Issue by TheKevJames** [TheKevJames/tools#989]
Support this, probably via the config.json? [Docs](https://cloud.google.com/pubsub/docs/ordering#rest)

While we're there, might be nice to extend support to "whatever additional/optional fields this API supports"! The API can be found [here](https://github.com/googleapis/python-pubsub/blob/7cf2d11ce20bf9080c4434b920bdf419faac5d88/google/pubsub_v1/services/subscriber/client.py#L740).

**Comment by shintasmith** [comment]
Yes, I would definitely love it if the SubscriptionConfig can also support this:
* enable_exactly_once_delivery
* ack_deadline_seconds

Thank you!

**Comment by TheKevJames** [comment]
Adding support for arbitrary config options (including both of those) in #1210 -- please feel free to test that branch (`quay.io/thekevjames/gcloud-pubsub-emulator:83a95e60a8`) and provide any feedback!
